app/Community/Components/UserProfileMeta.php

```diff
@@ -477,7 +477,7 @@ private function calculateRecentPointsEarned(User $user, string $preferredMode =
 
         foreach ($achievements as $playerAchievement) {
             $achievementDate = $playerAchievement->{$dateColumn};
-            $daysAgo = abs($now->diffInDays($achievementDate, false));
+            $daysAgo = abs((int) $now->diffInDays($achievementDate, false));
```

Comment:
If I'm reading the docs correctly, if the second parameter is `true`, an absolute value will be returned, so the `abs` can be eliminated by passing `true`.

Reply:
Updated in latest.